### ISSUE TITLE
Accuracy of UCCA conversion tool.

### ISSUE THREAD

**Issue by wangxinyu0922** [cfmrp/mtool#9]
Dear Organizers,

I'm interested in the [conversion tool](https://github.com/cfmrp/mtool/blob/master/ucca/convert.py) for ucca, I wonder whether it is something similar to the conversion tool in [SemEval 2019 task1](https://competitions.codalab.org/competitions/19160#learn_the_details-evaluation). If it is, I wonder what is the accuracy of it in converting the ucca MRP format to other formats (SDP, conll, ...). Can you provide some scores like the table in [SemEval 2019 task1](https://competitions.codalab.org/competitions/19160#learn_the_details-evaluation)?

Thank you.

**Comment by danielhers** [comment]
The converter between each format and the MRP format is supposed to preserve 100% of the information (@oepen, please confirm). We did not implement conversion among the different formalisms, though participants are welcome to do it if it helps with parsing. Differently from SemEval 2019 Task 1, here we expect participants to directly work with the MRP format rather than allowing submissions in any format for all formalisms.

**Comment by oepen** [comment]
well, depending on what one considers relevant information, it could be said that the current input codecs in mtool are lossy (by design).  for the 2019 shared task, we decided to only represent information in the MRP files that we actually want participating parsers to predict, and which therefore will be scored.  that led to us deliberately leaving out :wiki links in AMRs and most node properties in EDS (e.g. tense, aspect, number).  furthermore, there is some normalization that i would consider information-preserving, in the abstract, but that would inhibit coming back to the exact same textual form in a round-trip of, say, PenMan to MRP to PenMan; specifically, several of the frameworks (AMR, EDS, and UCCA) used node identifiers that are aimed to contribute to human readability, whereas in MRP we have adopted the position that node identifiers should just be integers.
